week-2/debug/recipe.js

```diff
@@ -10,6 +10,8 @@ const recipe = {
   ingredients: ["olive oil", "tomatoes", "salt", "pepper"],
 };
 
-console.log(`${recipe.title} serves ${recipe.serves}
-  ingredients:
-${recipe}`);
+console.log(`${recipe.title} serves ${recipe.serves} ingredients:
+`);
+ for(ingredients of recipe.ingredients){
+  console.log(ingredients)
```

Comment:
Nice spot that each ingredient should be on a new line, and great use of a `for...of` loop!